.github/CODEOWNERS

```diff
@@ -7,7 +7,7 @@
 # global owners are only requested if there isn't a more specific
 # codeowner specified below. For this reason, the global codeowners
 # are often repeated in package-level definitions.
-*       @evan-forbes @Wondertan
+    *     @evan-forbes @Wondertan
```

Comment:
reminder to myself that we should update these in a different PR

Reply:
I think they're updated in the spec change pr

Reply:
FYI the spec PR targets celestia-app not celestia-core